oneanddone/tasks/templates/tasks/emails/feedback_email.txt

```diff
@@ -5,3 +5,8 @@ The task: {{ task_name }} ({{ task_link }}) was {{ task_state }}.
 Feedback:
 
 {{ feedback }}
+
+Time spent on the task:
```

Comment:
Because the time spent will never be longer than 5 characters, I suggest just outputting it on the same line as this, such as:
```
Time spent on the task: {{ time_spent_on_task }}m
```